gtsam/sfm/sfm.i

```diff
@@ -310,4 +318,38 @@ class TranslationRecovery {
       const gtsam::BinaryMeasurementsUnit3& relativeTranslations) const;
 };
 
+namespace gtsfm {
+
+#include <gtsam/sfm/DsfTrackGenerator.h>
+
+class MatchIndicesMap {
+  MatchIndicesMap();
+  MatchIndicesMap(const gtsam::gtsfm::MatchIndicesMap& other);
+
+  size_t size() const;
+  bool empty() const;
+  void clear();
+  gtsam::gtsfm::CorrespondenceIndices at(const pair<size_t, size_t>& keypair) const;
```

Comment:
This line breaks the Matlab wrapper because `std::map` doesn't have an `at` method that takes a `std::pair` as its argument, rather it takes the key type.

[Docs](https://en.cppreference.com/w/cpp/container/map/at)

@dellaert there is no unit test for this.
@johnwlambert @akshay-krishnan reviewers need to do a better job. :slightly_smiling_face:

Reply:
My wrapper knowledge is very limited, sorry for missing that!

Reply:
No worries man. Just something to keep in mind going forward. :slightly_smiling_face:

Reply:
what would the fix be? just changing this to `const IndexPair& keypair`? 
if we do that, can we do `.at((i, j))` in. python? or will we need to construct `IndexPair(i, j)` ?